calibration/PARADK/CGINS-PARADK-20582__20220131.csv

```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20582,CC_dark_offset,1.31, source_file PARAD-K_QSP2200_SN_20582_Calibration_2022-01-31.pdf
+20582,CC_scale_wet,9.34E-18,
```

Comment:
good, with CC_dark_offset Probe Dark and CC_scale_wet units Volts/(quanta/cm^2sec)